Wordle.Api/Wordle.Api/Dtos/WordDateDTO.cs

```diff
@@ -0,0 +1,10 @@
+namespace Wordle.Api.Dtos;
+
+public record WordDateDTO
+{
+	public string Date { get; set; }
+	public int GameCount { get; set; }
+	public double AverageAttempts { get; set; }
+	public int AverageSeconds{ get; set; }
+    //public Array PlayerList{ get; set; }
+}
```

Comment:
I am not sure about this because Tim working on that feature.

Reply:
Yeah I believe it was because I based my work on the previous assignment whereas Sergei did it based on the classwork, and I didn't notice/realize until too late, and tbh I'd prefer to leave it as is since it's working lol